src/data/data-gig-detail.html

```diff
@@ -1,12 +1,18 @@
 <link rel="import" href="../../bower_components/polymer/polymer.html">
 
 <link rel="import" href="../../bower_components/polymerfire/firebase-document.html">
+<link rel="import" href="data-gigs-list.html">
 
 
 <dom-module id="data-gig-detail">
 <template>
   <firebase-document id="document" app-name="polygigs"
     path="/gigs/[[gigId]]" data="{{raw}}"></firebase-document>
+
+  <!--<template is="dom-if" if="{{!document.online}}">-->
```

Comment:
This is outdated code. I was doing it wrongly assuming I could reference `app-indexeddb-mirror` by accessing it by the assigned id of `document`. See updated code.